provisioning/provisioning-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/provisioning/service/auth/ProvisioningServiceSasTokenTest.java

```diff
@@ -66,12 +66,12 @@ public void constructorSucceeded() throws Exception
             final SecretKeySpec secretKeySpec;
             Mac mac;
             {
```

Comment:
Is this extra braces?

Reply:
it looks like an anonymous block, possibly to limit variable scope

Reply:
https://stackoverflow.com/questions/1563030/anonymous-code-blocks-in-java

Reply:
Do you see its usage here?

Reply:
I don't see any issue in the current test and refactoring the entire test doesn't seem worthwhile to me.

Reply:
It's ok if you don't want to refactor this part. It caught my attention as you modified the piece of code inside this block.